api-server/src/server/boot/certificate.js

```diff
@@ -68,7 +68,7 @@ export function getFallbackFullStackDate(completedChallenges, completedDate) {
 
   const latestCertDate = completedChallenges
     .filter(chal => chalIds.includes(chal.id))
-    .sort((a, b) => b.completedDate - a.completedDate)[0].completedDate;
+    .sort((a, b) => b.completedDate - a.completedDate)[0]?.completedDate;
```

Comment:
```suggestion
    .sort((a, b) => b.completedDate - a.completedDate)?.[0]?.completedDate;
```

Reply:
`.filter` and `.sort` return empty arrays on no result, right? When would it be undefined?

Reply:
This is fine as is.